src/handlers/express.ts

```diff
@@ -56,6 +56,7 @@ export function expressRequestHandler(
         // We push the current correlationId out as a header so that a TrackJS
         // agent on the client-side of the request can link up with us.
         res.setHeader("TrackJS-Correlation-Id", correlationId);
+        res.setHeader("Access-Control-Expose-Headers", "TrackJS-Correlation-Id");
```

Comment:
This would smash any other cors headers that were added somewhere else, correct? Maybe we can cross that bridge when we get to it.

Reply:
Bah, I didn't see eric's comment, I had an stale view of the PR.